infertrade/utilities/operations.py

```diff
@@ -318,5 +318,6 @@ def fit(self, X, y=None):
     def transform(self, X: pd.DataFrame, y=None):
         """Converts positions into the cumulative portfolio return."""
         X_ = deepcopy(X)
-        X_[PandasEnum.VALUATION.value] = calculate_portfolio_performance_python(X)[PandasEnum.VALUATION.value]
+        Y = deepcopy(X)
+        X_[PandasEnum.VALUATION.value] = calculate_portfolio_performance_python(Y)[PandasEnum.VALUATION.value]
         return X_
```

Comment:
This transform was unsafe, seen as calculate_portfolio_performance_python makes changes to the DataFrame it receives as a parameter.

Reply:
Good to change, but could use different new variable than Y, like "X_copy"? As y is in argument so could get confused.

Reply:
I have amended.